observations/obs_converters/even_sphere/README.rst

```diff
@@ -8,69 +8,113 @@ generate a vertical column of observations.  This could mimic
 a radiosonde observing network, for example.
 
 
-
-This directory contains a MATLAB script that generates
-input for the 'create_obs_sequence' program.  It takes
-a number of vertical levels and a total number of points,
-and generates a roughly evenly distributed set of observations
+This directory contains MATLAB scripts (even_sphere*.m) 
+that generate input for the `create_obs_sequence 
+<../../../assimilation_code/programs/create_obs_sequence/create_obs_sequence.html>`_ program.  
+They takes a number of vertical levels and a total number of points,
+and generate a roughly evenly distributed set of observations
 across the entire globe.  Note that the number of obs
-will be the number of points times the number of vertical
-levels.
+will be the number of points times the number of vertical levels.  
+``Even_sphere.m`` also takes observation error variances 
+and includes them in the observation templates.
+
+Here there are also scripts (run_fixed_network_*.csh) which use the
```

Comment:
again, the * is messing up the formatting.

Reply:
Added \ before *.